app/models/page_triple_paragraph_component.rb

```diff
@@ -0,0 +1,5 @@
+class PageTripleParagraphComponent < ApplicationRecord
```

Comment:
I am not married to this class name, I went with it because it is similar to the existing `PageParagraphComponent`